requirements.txt

```diff
@@ -284,3 +284,6 @@ Blinker==1.4 \
     --hash=sha256:471aee25f3992bd325afa3772f1063dbdbbca947a041b8b89466dc00d606f8b6
 Flask-Cache==0.13.1 \
     --hash=sha256:90126ca9bc063854ef8ee276e95d38b2b4ec8e45fd77d5751d37971ee27c7ef4
+structlog==16.1.0 \
+    --hash=sha256:a8cc45c208c6b251031ec223940552eccd97dbdb322474e0c4ec9e50f76a225a
+structlog[dev]==16.1.0
```

Comment:
Done, the library installed is actually colorama and structlog[dev] is just an alias to add that dependency on